AppHandling/Get-NavContainerAppInfo.ps1

```diff
@@ -15,30 +15,45 @@
   Specifies how (if any) you want to sort apps based on dependencies to other apps
  .Parameter publishedOnly
   Get published apps
+ .Parameter appFilePath
+  Specifies the path to a Business Central app package file (N.B. the path should be shared with the container)
  .Example
   Get-BcContainerAppInfo -containerName test2
  .Example
   Get-BcContainerAppInfo -containerName test2 -tenant mytenant -tenantSpecificProperties
  .Example
   Get-BcContainerAppInfo -containerName test2 -symbolsOnly
+ .Example
+  Get-BcContainerAppInfo -containerName test2 -appFilePath "C:\ProgramData\BcContainerHelper\Extensions\apx-dev\myApp.app"
 #>
 function Get-BcContainerAppInfo {
     Param (
         [string] $containerName = $bcContainerHelperConfig.defaultContainerName,
+        [Parameter(Mandatory = $false, ParameterSetName = 'Tenant')]
         [string] $tenant = "",
+        [Parameter(Mandatory = $false, ParameterSetName = 'AppFile')]
+        [string] $appFilePath,
+        [Parameter(ParameterSetName = 'SymbolsOnly')]
         [switch] $symbolsOnly,
+        [Parameter(ParameterSetName = 'Tenant')]
         [switch] $tenantSpecificProperties,
+        [Parameter(ParameterSetName = 'Tenant')]
         [ValidateSet('None','DependenciesFirst','DependenciesLast')]
         [string] $sort = 'None',
+        [Parameter(ParameterSetName = 'Tenant')]
         [switch] $publishedOnly,
+        [Parameter(ParameterSetName = 'Tenant')]
         [switch] $installedOnly
     )
 
 $telemetryScope = InitTelemetryScope -name $MyInvocation.InvocationName -parameterValues $PSBoundParameters -includeParameters @()
 try {
 
     $args = @{}
-    if ($symbolsOnly) {
+    if ($appFilePath) {
+        $args += @{ "Path" = $appFilePath }

```

Comment:
I think you need to use 
"Path" = Get-BcContainerPath $appFilePath
in order for this to work (unless the path is the same inside and outside the container)

Reply:
I think you might be right, I think I may have tested with a scenario where it works regardless.
I have made adjustments accordingly.